src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -324,6 +324,12 @@ struct uct_ib_mlx5_cmd_hca_cap_bits {
     uint8_t    reserved_at_328[0x3];
     uint8_t    log_max_pd[0x5];
     uint8_t    reserved_at_330[0xb];
+    uint8_t    dp_ordering_ooo_all_ud[0x1];
```

Comment:
this seems wrong, we need to remove the line 326 `uint8_t reserved_at_330[0xb]` I think

Reply:
Wow, thanks, missed that